### PR TITLE
Adds flutter v1 release blog article

### PR DESCRIPTION
**Pull request by dshukertjr**
- Adds supabase-flutter v1 release blog article
- Swaps the main image of supabase-flutter v1 developer preview article to match other blog article's theme

### CODE REVIEW

File: apps/www/_blog/2022-10-21-supabase-flutter-sdk-v1-released.mdx [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+title: 'supabase-flutter v1 Released'
+description: We've released supabase-flutter v1. TODO write more
```
Comment (RamiroND) [inline apps/www/_blog/2022-10-21-supabase-flutter-sdk-v1-released.mdx:3]: Remove TODO and add two or three of the new things.

Comment (dshukertjr) [reply apps/www/_blog/2022-10-21-supabase-flutter-sdk-v1-released.mdx:3]: Oops! Thanks for catching this one!

File: apps/www/_blog/2022-10-21-supabase-flutter-sdk-v1-released.mdx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+// subscribe to the above changes
+channel.subscribe((status) async {
+  if (status == 'SUBSCRIBED') {
+    // if subscribed successfully, send presence event
+    final status = await channel.track({'user_id': myUserId});
+  }
+});
+```
+
+These are just tip of the ice burg of all the updates that we shipped in v1. Check out the [documentation](https://supabase.com/docs/reference/dart/) to see the full list of what updates supabase-flutter can do.
```
Comment (RamiroND) [inline apps/www/_blog/2022-10-21-supabase-flutter-sdk-v1-released.mdx:92]: iceberg

**Comment by dshukertjr** [comment]
@RamiroND Thanks for catching them! Fixed now.

File: apps/www/_blog/2022-10-21-supabase-flutter-sdk-v1-released.mdx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  if (status == 'SUBSCRIBED') {
+    // if subscribed successfully, send presence event
+    final status = await channel.track({'user_id': myUserId});
+  }
+});
+```
+
+These are just tip of the iceburg of all the updates that we shipped in v1. Check out the [documentation](https://supabase.com/docs/reference/dart/) to see the full list of what updates supabase-flutter can do.
+
+## Acknoledgements
```
Comment (stavares843) [inline apps/www/_blog/2022-10-21-supabase-flutter-sdk-v1-released.mdx:94]: minor typo - Acknoledgements instead of Acknowledgements

Comment (dshukertjr) [reply apps/www/_blog/2022-10-21-supabase-flutter-sdk-v1-released.mdx:94]: Thanks for catching this one!
```suggestion
## Acknowledgements
```

File: apps/www/_blog/2022-10-21-supabase-flutter-sdk-v1-released.mdx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+// subscribe to the above changes
+channel.subscribe((status) async {
+  if (status == 'SUBSCRIBED') {
+    // if subscribed successfully, send presence event
+    final status = await channel.track({'user_id': myUserId});
+  }
+});
+```
+
+These are just tip of the iceburg of all the updates that we shipped in v1. Check out the [documentation](https://supabase.com/docs/reference/dart/) to see the full list of what updates supabase-flutter can do.
```
Comment (stavares843) [inline apps/www/_blog/2022-10-21-supabase-flutter-sdk-v1-released.mdx:92]: @RamiroND mentioned iceberg previously here also 🔨

Comment (dshukertjr) [reply apps/www/_blog/2022-10-21-supabase-flutter-sdk-v1-released.mdx:92]: Ah, I really can't spell, can I ? 😂

Comment (dshukertjr) [reply apps/www/_blog/2022-10-21-supabase-flutter-sdk-v1-released.mdx:92]:
```suggestion
These are just tip of the iceberg of all the updates that we shipped in v1. Check out the [documentation](https://supabase.com/docs/reference/dart/) to see the full list of what updates supabase-flutter can do.
```

**Review by RamiroND — APPROVED** [review]

**Pull request merged**